spec/8-service-apis.md

```diff
@@ -1,15 +1,17 @@
 # 8 Service APIs
 
-This section describes external APIs that must be implemented by the building block. Additional APIs may be implemented by the building block (all APIs must adhere to the standards and protocols defined), but the listed APIs define a minimal set that must be provided by any implementation.
+This section describes external APIs that must be implemented by the building block. Additional APIs may be implemented by the building block (all APIs must adhere to the standards and protocols defined), but the listed APIs define a minimal set that must be provided by any implementation. 
 
-{% hint style="success" %}
-All APIs will be defined using the OpenAPI (Swagger) standard. The API definitions will be hosted outside of this document. This section may provide a brief description of required APIs.&#x20;
+The current API specification can be found here:
 
-This section will primarily contain links to the GitHub repository for OpenAPI definition (yaml) files as well as to a website hosted by GovStack that provides a live API documentation portal.
+* [API and Data model main definition document](https://docs.google.com/spreadsheets/d/1snIszqyTGYk1u25liwQ_1jONTsQeH7D8aqv1Td74xt4/edit?usp=sharing) _(for commenting)_
+* [Pull-Request for 0.8 API spec](https://github.com/GovStackWorkingGroup/BuildingBlockAPI/pull/15) _(Raise PR if you want to suggest a change)_
+* [Latest version rendered API spec](https://app.swaggerhub.com/apis/GovStack/consent-management-bb) _(view only)_
```

Comment:
There is actually a really fab way to display OpenAPI specs right inside the specs. At https://docs.govstack.global/building-blocks/information-mediation/10-service-apis You can see the output of what is written at https://github.com/GovStackWorkingGroup/specifications/blob/main/building-blocks/information-mediation/10-service-apis.md
Obviosuly, this is easier to do via the UI once we wire this up

Reply:
Will raise an issue for this and get this sorted.